packages/api/test/pin.spec.js

```diff
@@ -279,9 +289,27 @@ describe('Pinning APIs endpoints', () => {
       assert.strictEqual(data.count, 6)
       assert.strictEqual(data.results.length, 3)
     })
+
+    it('error if user not authorized to pin', async () => {
+      const notAuthorizedToken = await getTestJWT('test-upload', 'test-upload')
+      const res = await fetch(new URL('pins', endpoint).toString(), {
+        method: 'GET',
+        headers: {
+          Authorization: `Bearer ${notAuthorizedToken}`,
+          'Content-Type': 'application/json'
+        }
+      })
+      assert(!res.ok)
+      assert.strictEqual(res.status, (new PinningUnauthorizedError()).status)
```

Comment:
Suggestion: parse the response JSON and check the `code` is `PinningUnauthorizedError.CODE`.

...and repeat for all tests checking this :)

Reply:
Yep that's cleaner. Changed here and elsewhere.